conf/server.properties

```diff
@@ -0,0 +1,50 @@
+#Minecraft server properties
+#Sat Sep 11 17:53:29 GMT 2021
+enable-jmx-monitoring=false
+rcon.port=25575
+gamemode=survival
+enable-command-block=false
+enable-query=false
+level-name=world
+motd=Pingus playhouse
+query.port=25565
+pvp=true
+difficulty=hard
+network-compression-threshold=256
+require-resource-pack=false
+max-tick-time=60000
+use-native-transport=true
+max-players=20
+online-mode=true
+enable-status=true
+allow-flight=true
+broadcast-rcon-to-ops=true
+view-distance=10
+server-ip=
+resource-pack-prompt=
+allow-nether=true
+server-port=25565
+enable-rcon=true
+sync-chunk-writes=true
+op-permission-level=4
+prevent-proxy-connections=false
+resource-pack=
+entity-broadcast-range-percentage=100
+rcon.password=skt
+player-idle-timeout=0
+debug=false
+force-gamemode=false
+rate-limit=0
+hardcore=false
+white-list=false
+broadcast-console-to-ops=true
+spawn-npcs=true
+spawn-animals=true
+snooper-enabled=true
+function-permission-level=2
+text-filtering-config=
+spawn-monsters=true
+enforce-whitelist=false
+resource-pack-sha1=
+spawn-protection=0
```

Comment:
should probably set to 1

Reply:
Yeah, I'll update it
Apparently it protects a 2x+n square, and not n chunks as I thought